src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -145,10 +145,26 @@ public void addAuthor(Author author, List<String> ignoreGlobList) {
     public void removeIgnoredAuthors(List<String> ignoredAuthorsList) {
         for (String author : ignoredAuthorsList) {
             if (authorEmailsAndAliasesMap.containsKey(author)) {
-                authorList.remove(authorEmailsAndAliasesMap.get(author));
+                removeIgnoredAuthorsInformation(author);
             }
         }
-        resetAuthorInformation();
+    }
+
+    /**
+     * Removes all information of the {@code author} from the configs
```

Comment:
I think the Javadoc should say whether `author` is a git ID, email, alias (or any of these)

Reply:
Done.